sklearn/ensemble/gradient_boosting_cv.py

```diff
@@ -0,0 +1,804 @@
+"""
+This module contains Gradient Boosting Classifier and Regressor with
+Cross Validation.
+"""
+
+import warnings
+import numpy as np
+
+from ..base import BaseEstimator
+from ..metrics.scorer import check_scoring
+from ..utils.validation import check_X_y
+
+from .gradient_boosting import GradientBoostingClassifier
+from .gradient_boosting import GradientBoostingRegressor
+from ..tree._tree import DTYPE
+
+from ..model_selection import check_cv
+from ..model_selection import ParameterGrid
+from ..model_selection import ParameterSampler
+from ..model_selection._search import _CVScoreTuple
+
+from ..externals.joblib import Parallel, delayed
+
+
+class GradientBoostingClassifierCV(BaseEstimator):
+    """ Gradient Boosting Classification with Cross Validation
+
+    This class keeps adding estimators to the underlying GB estimator till
```

Comment:
I would rephrase this paragraph to first about what it does, not how it works. This is efficient cross-validation for GradientBoosting.